src/integrationTest/java/org/opentripplanner/base/LegGeometryIT.java

```diff
@@ -0,0 +1,88 @@
+package org.opentripplanner.base;
+
+import com.google.common.collect.ImmutableList;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.locationtech.jts.geom.Coordinate;
+import org.opentripplanner.IntegrationTest;
+import org.opentripplanner.api.model.Itinerary;
+import org.opentripplanner.api.model.Leg;
+import org.opentripplanner.api.resource.Response;
+import org.opentripplanner.util.PolylineEncoder;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class LegGeometryIT extends IntegrationTest {
+
+    @Test
+    public void testLegGeometryContinuity() {
+        List<String> destinations = ImmutableList.of("53.134889, 18.035789", "53.142835, 18.018029", "53.130221, 18.039692", "53.111929, 18.035472", "53.120262, 17.977395");
```

Comment:
To są losowe punkty czy jakieś konkretne?

Reply:
losowe ulice w Bydgoszczy w różnych częściach miasta